### PR TITLE
Red Card? How to test. RKI poster

### PR DESCRIPTION
**Pull request by dsarkar**

### CODE REVIEW

**Comment by dsarkar** [comment]
DE

image

**Comment by dsarkar** [comment]
EN

image

File: src/data/faq_de.json [hunk 1]
```diff
@@ -160,6 +160,16 @@
                             "in die Corona-Warn-App ein."
                         ]
                     },
+                    {
+                        "title": "Rote Karte? So geht es zum Corona Test",
+                        "anchor": "red_card_how_to_test",
+                        "active": true,
+                        "textblock": [
+                            "Wenn die Corona-Warn-App Ihnen mit der roten Karte ein hohes Infektionsrisiko anzeigt, isolieren Sie sich bitte zügig und konsultieren Sie einen Arzt, um sich ggf. testen zu lassen.",
+                            "Für einen reibungslosen Ablauf beim Testen, zum Empfang des Testbefundes und zu Warnung Anderer bei einem positiven Ergebnis folgen Sie bitte den Schritten in <a href='../../assets/documents/BPA_Corona-Warn-App_RKI-Plakat_Testcenter_A1_ICv2_RZ02_x3.pdf' target='_blank' rel='noopener'>dieser Übersicht vom Robert-Koch-Institut</a>.",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:169]: Robert-Koch-Institut => Robert Koch-Institut

Reference: Bottom of https://www.rki.de/DE/Home/homepage_node.html
"Das Robert Koch-Institut ist ein Bundesinstitut im Geschäftsbereich des Bundesministeriums für Gesundheit"

File: src/data/faq_de.json [hunk 2]
```diff
@@ -160,6 +160,16 @@
...
                         ]
                     },
+                    {
+                        "title": "Rote Karte? So geht es zum Corona Test",
+                        "anchor": "red_card_how_to_test",
+                        "active": true,
+                        "textblock": [
+                            "Wenn die Corona-Warn-App Ihnen mit der roten Karte ein hohes Infektionsrisiko anzeigt, isolieren Sie sich bitte zügig und konsultieren Sie einen Arzt, um sich ggf. testen zu lassen.",
+                            "Für einen reibungslosen Ablauf beim Testen, zum Empfang des Testbefundes und zu Warnung Anderer bei einem positiven Ergebnis folgen Sie bitte den Schritten in <a href='../../assets/documents/BPA_Corona-Warn-App_RKI-Plakat_Testcenter_A1_ICv2_RZ02_x3.pdf' target='_blank' rel='noopener'>dieser Übersicht vom Robert-Koch-Institut</a>.",
+                            "<a href='../../assets/documents/BPA_Corona-Warn-App_RKI-Plakat_Testcenter_A1_ICv2_RZ02_x3.pdf' target='_blank' rel='noopener'><img src='../../assets/documents/BPA_Corona-Warn-App_RKI-Plakat_Testcenter_A1_ICv2_RZ04_x3b.jpg' alt='PDF'></a>"
```
Comment (MikeMcC399) [inline src/data/faq_de.json:170]: Link to
BPA_Corona-Warn-App_RKI-Plakat_Testcenter_A1_ICv2_RZ02_x3.pdf does not work. The document is
BPA_Corona-Warn-App_RKI-Plakat_Testcenter_A1_ICv2_RZ04_x3b.pdf

File: src/data/faq.json [hunk 3]
```diff
@@ -159,6 +159,16 @@
                             "<ul><li>'NEXT STEPS' -> 'Enter TAN'</li></ul>"
                         ]
                     },
+                    {
+                        "title": "Red Card? How to test",
```
Comment (MikeMcC399) [inline src/data/faq.json:163]: "How to test" sounds like it is about testing the Red Card, not the person. Also the FAQ article more about sharing the result than about testing.

Suggest instead:
"Red Card? How to get tested and best use the app"

File: src/data/faq.json [hunk 4]
```diff
@@ -159,6 +159,16 @@
                             "<ul><li>'NEXT STEPS' -> 'Enter TAN'</li></ul>"
                         ]
                     },
+                    {
+                        "title": "Red Card? How to test",
+                        "anchor": "red_card_how_to_test",
+                        "active": true,
+                        "textblock": [
+                            "If the Corona Warn app indicates a high infection risk with the red card, please isolate yourself quickly and consult a doctor to get screened if necessary.",
```
Comment (MikeMcC399) [inline src/data/faq.json:167]: Corona Warn app => Corona-Warn-App

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
- The German document links are wrong and definitely need to be fixed.
- I've marked the app name and RKI name for alignment with other uses of the names.
- Also I suggested a different title for the English FAQ which is more like a translation of the German title.

**Review by mtb77 — CHANGES_REQUESTED** [review]
Please have a look at the suggested changes

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks!

**Pull request merged**